tsdb/db_test.go

```diff
@@ -1176,11 +1176,11 @@ func TestSizeRetention(t *testing.T) {
 	testutil.Ok(t, err)
 	// Expected size should take into account block size + WAL size
 	expSize = blockSize + walSize
-	actRetentCount := int(prom_testutil.ToFloat64(db.metrics.sizeRetentionCount))
+	actRetentionCount := int(prom_testutil.ToFloat64(db.metrics.sizeRetentionCount))
```

Comment:
This seems to be less of a spelling fix and more writing out an intentionally abbreviated term - but I like the new version more :)